lib/common/db-renderable-content.js

```diff
@@ -118,6 +120,11 @@ function dbRenderableContentServiceFactory(
 
     DbRenderableContentService.prototype.render = function render (name, options, scope) {
         return this.get(name, scope).then(function (template) {
+            if (!template) {
```

Comment:
I called the variable 'template', but it can be a template, profile or view.  Should we be throwing a TemplateRenderError when it can be either of the three?  Maybe we just want to use asserts and then let the parent object handle throwing the more specific error (if it is needed) or use the collectionName to throw the most correct collection specific error?